controllers/controller.js

```diff
@@ -11,4 +11,15 @@ apiController = (request, response) => {
   response.status(200).send(endPoints);
 };
 
-module.exports = { topicsController, apiController };
+articleIdController = (request, response, next) => {
+  const { article_id } = request.params;
+  articleIdModel(article_id)
+    .then((result) => {
+      response.status(200).send(result);
```

Comment:
Please use a descriptive name, e.g. "article" for the variable returned by articleIdModel.  Then when sending this please use as the key on an object eg. {article}.  This will mean that the response will consistently be on the body as a property and can be de-structured in the test file as body.article.